src/Compilers/VisualBasic/Portable/Syntax/Syntax.xml

```diff
@@ -5084,25 +5084,43 @@ Unify Enum declarations with other block type declarations.
     <!--********************
       -  TupleElementSyntax
       **********************-->
-    <node-structure name="TupleElementSyntax" parent="VisualBasicSyntaxNode">
+    <node-structure name="TupleElementSyntax" abstract="true" parent="VisualBasicSyntaxNode">
       <description>Represents a single declaration of a tuple element. </description>
       <grammar>TupleElement</grammar>
+    </node-structure>
 
-      <node-kind name="TupleElement"></node-kind>
+    <!--********************
+      -  TypedTupleElementSyntax
+      **********************-->
+    <node-structure name="TypedTupleElementSyntax" parent="TupleElementSyntax">
+      <description>Represents a single declaration of a tuple element supplying only the type. </description>
+      <grammar>TypedTupleElement</grammar>
 
-      <child name="IdentifierName" optional="true" kind="IdentifierName">
-        <description>An identifier naming a tuple element.</description>
+      <node-kind name="TypedTupleElement"></node-kind>
+
+      <child name="Type" optional="true" kind="@TypeSyntax">
```

Comment:
Right, that is the whole point for the node

---
In reply to: [82260999](https://github.com/dotnet/roslyn/pull/14284#discussion_r82260999) [](ancestors = 82260999)